src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Draw, Area {
```

Comment:
The class name 'Figure' should not be a verb. Consider renaming it to something that represents an entity or concept, such as 'Shape' or 'AbstractFigure' if you intend to keep it abstract.